app/models/claim.rb

```diff
@@ -35,15 +35,16 @@ class Claim < ApplicationRecord
     updated_at: false,
     verified_fields: false,
     verify_response: true,
-    approved_at: false,
-    approved_by: false,
+    check_id: false,
   }.freeze
 
   enum student_loan_country: StudentLoans::COUNTRIES
   enum student_loan_start_date: StudentLoans::COURSE_START_DATES
   enum student_loan_courses: {one_course: 0, two_or_more_courses: 1}
   enum student_loan_plan: STUDENT_LOAN_PLAN_OPTIONS
 
+  belongs_to :check, optional: true
```

Comment:
I think this should actually be a `has_many` relationship, given the discussion about replacing checks if the status changes, the most recently created check being the "current" one.

Reply:
Managed to sort this. It may need some tweaking/rebasing (and we may need to look at a state machine at some point (but not in the tranche of work)), but the hard work is done.